internal/cli/command/cluster/create.go

```diff
@@ -458,6 +465,27 @@ func getProviders() map[string]interface{} {
 				Rbac:    true,
 			},
 		},
+		pkeOnVsphere: pipeline.CreatePkeOnVsphereClusterRequest{
+			Type: pkeOnVsphere,
+			Kubernetes: pipeline.CreatePkeClusterKubernetes{
+				Version: "1.15.3",
+				Rbac:    true,
+			},
+			Folder:       "folder",
+			Datastore:    "DatastoreCluster",
+			ResourcePool: "resource-pool",
+			NodePools: []pipeline.PkeOnVsphereNodePool{
+				{
+					Name:          "master",
+					Roles:         []string{"master", "worker"},
```

Comment:
Shouldn't we recommend here to create a separate master & worker pool?

Reply:
we don't do for other providers either. single-node is fine for first try imo